omc3/optics_measurements/kick.py

```diff
@@ -98,10 +100,16 @@ def _get_action(meas_input, lin: pd.DataFrame, plane: str) -> np.ndarray:
 
     if meas_input.accelerator.excitation:
         amps = frame.loc[:, f"{AMPLITUDE}{plane}"].to_numpy()
-        err_amps = frame.loc[:, f"{ERR}{AMPLITUDE}{plane}"].to_numpy()
+        try:
```

Comment:
Can we add a small comment here to specify we expect the columns might not be there becayse they only show up if we call cleaning? (is that correct?)